app/services/moab_on_storage.rb

```diff
@@ -1,8 +1,7 @@
 # frozen_string_literal: true
 
-##
-# Utility methods for Moab::StorageObject and Druid
-class MoabUtils
+# Services for a Moab on storage (as opposed to a moab in the catalog)
```

Comment:
"catalog" --> db ?

or some variant of:
"distinguish between the moab record in the db and the Moab on local storage (and the zipped moab versions replicated to the cloud)" 

?